client/app/dedicatedCloud/datacenter/drp/steps/ovh/dedicatedCloud-datacenter-drp-ovh.html

```diff
@@ -0,0 +1,45 @@
+<div data-ovh-alert="dedicatedCloudDatacenterDrpAlert"></div>
+
+<oui-message data-ng-if="$ctrl.drpInformations.state === 'toProvision' || $ctrl.drpInformations.state === 'provisionning' || $ctrl.isEnabling"
+             data-type="info">
+    <p data-translate="dedicatedCloud_datacenter_drp_confirm_creation_pending"></p>
+    <p data-translate="dedicatedCloud_datacenter_drp_confirm_creation_pending_task"></p>
+</oui-message>
+
+<div class="oui-progress-tracker my-5">
```

Comment:
Wouldn't a `<oui-progress>` can fit your needs?

Reply:
I need this : https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/progress-tracker, not this https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/progress ;)